### DISCUSSION TITLE
Pasting a value/argument after a trigger

### DISCUSSION THREAD

**Discussion by geekscrapy** [espanso/espanso#1617] [Feature requests and Ideas]
1. Can I have a trigger like this? Using a space after the initial `:h`

```yml
  - regex: ":h (?P<hash>[a-f0-9]{32})"
    replace: "File hash: ({{hash}})"
```

2. A hash is quite long so I want to paste it rather than type it. But if I paste it, it doesn't trigger. If I type it, it does trigger. Is this a known limitation?

**Comment by jwigg-netscout** [comment]
1. Yes, you can have spaces in triggers
2. This is a known limitation. To achieve the result you're looking for, you'd make use of the form plugin, which would display an input box that you could paste the hash into, and then it would complete the expansion. That said, given that you are pasting the hash, it seems like the entirety of the actual expansion i just expanding ":h" -> "File hash: ", and you could simply paste the hash at the end of the text.

**Reply by geekscrapy** [reply]
Thanks

The example was just a dummy. In actual fact I wanted to use the hash as part of a python command which does a lookup against an API and returns other data.

I'll have a look at the form plugin, thanks!
